.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
It should be build/_output only because build have the docker and push file also